src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -634,6 +634,16 @@ public function text(string $default = null, bool $normalizeWhitespace = true)
         return $text;
     }
 
+    /**
+     * Returns only the inner text that is the direct descendent of the current node, excluding any child nodes.
+     *
+     * @return string Inner text
+     */
+    public function innerText(): string
+    {
+        return $this->filterXPath('.//text()')->text();
```

Comment:
This can also be expressed as `child::node()/text()`. I don't know which is "better".